apps/api/src/app/widgets/usecases/mark-message-as/mark-message-as.usecase.ts

```diff
@@ -56,6 +56,7 @@ export class MarkMessageAs {
     });
 
     if (command.mark.seen != null) {
+      await this.sendAnalyticsEventForInviteTeamNudge(messages);
```

Comment:
ahh ok, got it... I was thinking that we want to call it whenever someone marks the message as seen... but we want this to be called in case of clicking in our NC Widget